### DISCUSSION TITLE
Zotero image notes output to Obsidian

### DISCUSSION THREAD

**Discussion by nadesss** [BryanJenksCommunity/FAQ#166] [Q&A]
Hi , Bryan:
I' m using your template to import note from Zotero using zotero plugin 'mdnotes' according to your lattest zotero video
the problem is I can't get image note imported with the text notes. Is there anything that can improve this workflow.
ps: I watched Danny's video about obsidian plugin 'Zotero Intergration', but that's another workflow, I think that compares nothing to yours. Thank you

**Comment by tallguyjenks** [comment]
like images from zotero papers that have been highlighted getting sent to obsidian? not quite sure what you're asking about

checking out the plugin danny uses is actually on my TODO list, just haven't gotten around to it yet!
